### PR TITLE
wifi: Fix secure associations

### PR DESCRIPTION
**Pull request by krish2718**
Update sdk-hostap with fixes for secure association.

Signed-off-by: Krishna T <krishna.t@nordicsemi.no>

### CODE REVIEW

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
@krish2718 It looks like the WiFi sample does not build with this `sdk-hostap` revision:
```
/jenkins_workspace/workspace/latest_sdk-nrf_PR-8218/modules/lib/hostap/src/l2_packet/l2_packet_zephyr.c
../../../../../modules/lib/hostap/src/l2_packet/l2_packet_zephyr.c: In function 'l2_packet_receive':
../../../../../modules/lib/hostap/src/l2_packet/l2_packet_zephyr.c:100:6: error: assignment discards 'const' qualifier from pointer target type [-Werror=discarded-qualifiers]
  100 |  hdr = (const struct ieee802_1x_hdr *) buf;
      |      ^
cc1: all warnings being treated as errors
```

**Comment by krish2718** [comment]
Raised https://github.com/nrfconnect/sdk-hostap/pull/10

File: boards/shields/nrf7002_ek/nrf7002_ek.overlay [hunk 1]
```diff
@@ -4,6 +4,10 @@
  */
 #include <freq.h>
 
+&uart1 {
+    status = "disabled";
+};
+
```
Comment (anangl) [inline boards/shields/nrf7002_ek/nrf7002_ek.overlay:10]: Was this change intentional?

Comment (krish2718) [reply boards/shields/nrf7002_ek/nrf7002_ek.overlay]: oops, fixed it. Thanks.

**Review by anangl — APPROVED** [review]
